### PR TITLE
Ratings model, table and migrations Features/8885

### PR DESCRIPTION
**Pull request by FuriousLlama**
# Fixes or Implements [AB#8885](https://qslvic.visualstudio.com/304a1f8c-dace-4f85-adf3-bf563d5b3a39/_workitems/edit/8885)

* [x] Enhancement
* [ ] Bug
* [ ] Documentation

## Reviewer Guidelines

A reviewer should examine the PR if the Status above indicates the PR is **Ready**.
Once ready, the reviewer should read the information below as well as supporting information like the Azure ticket to ensure they have a good understanding of the request.

The reviewer should then:

1. Is the PR template complete and accurate?

    Confirm by reading all sections below.

1. Confirm the merge branch

    A normal feature/bug branch should merge into the Dev branch.  Exceptions for hotfix and sprint merges do occur and the target would then be Master.

1. Do the code commits have an appropriate comment and include linkage to the azure ticket by including a reference to "AB#nnnn"? 

1. Review the Code Changes

    1. Do the changes match the description?

        The code commits should reflect the documented description and nothing more.  If additional changes are included the reviewer should call these out and request that the developer document the changes in Azure and in the description or remove the changed code.

    1. useful tips

        * Any obvious errors or omissions?
        * Does the code match the style of other similar work?
        * Is the code easy to understand?
        * Is the code well documented?
        * DB Migrations are appropriate?

If the Answer to any of the questions above is No, then a request for change should be the result of the review.

## Description

Created model and migrations for Ratings

## Testing

* [ ] Unit Tests Updated
* [ ] Functional Tests Updated
* [x] Not Required

### CODE REVIEW

File: Apps/Database/src/Context/GatewayDbContext.cs [hunk 1]
```diff
@@ -70,6 +70,7 @@ public GatewayDbContext(DbContextOptions<GatewayDbContext> options)
         public DbSet<Comment> Comment { get; set; } = null!;
         public DbSet<Communication> Communication { get; set; } = null!;
         public DbSet<CommunicationEmail> CommunicationEmail { get; set; } = null!;
+        public DbSet<Ratings> Ratings { get; set; } = null!;
```
Comment (leobcgov) [inline Apps/Database/src/Context/GatewayDbContext.cs:73]: I think we are using table name without "s". Ratings --> Rating

File: Apps/Database/src/Models/Ratings.cs [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+namespace HealthGateway.Database.Models
+{
+    using System;
+    using System.ComponentModel.DataAnnotations.Schema;
+
+#pragma warning disable CS1591 // self explanatory simple model
+#pragma warning disable SA1600 // self explanatory simple model
+    public class Ratings : AuditableEntity
+    {
+        [Column("RatingsId")]
```
Comment (leobcgov) [inline Apps/Database/src/Models/Ratings.cs:25]: DB Table name update to Rating will require column name update as well. RatingsId --> Rating

File: Apps/Database/src/Context/GatewayDbContext.cs (hunk 1 above)
Comment (FuriousLlama) [reply Apps/Database/src/Context/GatewayDbContext.cs:73]: singular vs plural, sure

File: Apps/Database/src/Migrations/20200817213421_AddRatingsTable.cs [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+// limitations under the License.
+//-------------------------------------------------------------------------
+// <auto-generated />
+#pragma warning disable CS1591
+using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace HealthGateway.Database.Migrations
+{
+    public partial class AddRatingsTable : Migration
```
Comment (leobcgov) [inline Apps/Database/src/Migrations/20200817213421_AddRatingsTable.cs:23]: Ratings --> Rating for here and the name of the migration scripts (20200817213421_AddRatingsTable.cs  and .designer..) as well if you don't mind to change :-)

Comment (FuriousLlama) [reply Apps/Database/src/Migrations/20200817213421_AddRatingsTable.cs:23]: I don't think is necessary, we have other names like that for the migrations

Comment (leobcgov) [reply Apps/Database/src/Migrations/20200817213421_AddRatingsTable.cs:23]: no need to change per our rocket chat :)

File: Apps/Database/src/Models/Rating.cs [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+#pragma warning disable CS1591 // self explanatory simple model
+#pragma warning disable SA1600 // self explanatory simple model
+    public class Rating : AuditableEntity
+    {
+        [Column("RatingId")]
+        public Guid Id { get; set; }
+
+        public int RatingValue { get; set; }
+
+        public bool Skip { get; set; }
```
Comment (leobcgov) [inline Apps/Database/src/Models/Rating.cs:30]: forgot to ask for Documentation of these props :-)

Comment (FuriousLlama) [reply Apps/Database/src/Models/Rating.cs:30]: we usually skip those.

**Pull request merged**